app/javascript/comments/Comments.vue

```diff
@@ -5,7 +5,7 @@
   template(v-if="store.currentUser")
     CommentForm(@submit="store.addComment")
   template(v-else)
-    p #[GoogleLoginButton.google-login-form.dark-mode-supported(:origin="windowLocationWithHash('comments')")] to add a comment.
```

Comment:
(Mostly unrelated change; I noticed when copying from here that this `.google-login-form` class is no longer needed, since this change: https://github.com/davidrunger/david_runger/pull/6599/files#diff-990c373ac35fdfc6f3e009894c7e0c44276f98f0bf472dcb50d9bbc0b9a31e28L53-L55 .)